.github/workflows/test-ubuntu.yml

```diff
@@ -1,16 +1,28 @@
 name: Python Tests
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [ master ]
```

Comment:
I think this may be why it's not running when you push to your branch.

Reply:
no it didn't run before either, i changed it to test if it made a difference.